screens/auth/WelcomeScreen.js

```diff
@@ -34,12 +36,23 @@ export default class WelcomeScreen extends React.Component {
           onPress={() => this.navigateSignup()}>
           <ButtonLabel color="white">Sign up</ButtonLabel>
         </FilledButtonContainer>
-        <WelcomeLogInContainer>
-          <Body color={Colors.secondaryText}>Already have an account?</Body>
-          <ButtonContainer onPress={() => this.navigateLogIn()}>
-            <ButtonLabel color={Colors.primaryGreen}>Log In</ButtonLabel>
-          </ButtonContainer>
-        </WelcomeLogInContainer>
+        <FilledButtonContainer
+          style={{ marginTop: 12 }}
+          color={Colors.lighterGreen}
+          width="100%"
+          onPress={() => this.navigateLogIn()}>
+          <ButtonLabel color="white">Log In</ButtonLabel>
```

Comment:
Also replace this with the appropriate color from `Colors.js` (I think it's `Colors.lightest`)

Reply:
Change 'white' to `Colors.lightest`